src/components/PublicKeyTweet/index.tsx

```diff
@@ -10,7 +10,7 @@ const PublicKeyTweet: React.FC = ({
   message,
   publicKey,
 }) => {
-  const publicKeyUrl = `https://proofof.dog/addr?dogecoin=${publicKey}`
+  const publicKeyUrl = `https://proofof.dog/addr?dogname=${dogname}&dogecoin=${publicKey}`
```

Comment:
reorder and ensure data are clean. 

- dogecoin=
- twitter= (lowercase, whitespace trimmed, @ symbol trimmed)
- dogname= (whitespace trimmed and truncated to max length 64)

Reply:
trim twitter to max length 64 also